<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget { 
border-image: url(:/icons/src/icons/main_background.png) 0 0 0 0 stretch stretch;
}

QLabel#app_icon {
background-color: rgb(0, 0, 0);
}

QLabel#suite_id {
padding-right: 10px;
}

QLabel#fixer_text {
padding-right: 10px;
}

QFrame#suite_modal {
background-color: rgb(255, 255, 255);
}

QFrame#fixer_modal {
background-color: rgb(255, 255, 255);
}

QPushButton#runner_btn {
background-color: rgb(54, 122, 255);
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>92</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sidebar">
     <item>
      <widget class="QPushButton" name="ui_icon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/src/icons/ui-icon.png</normaloff>:/icons/src/icons/ui-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>68</width>
         <height>68</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dhcp_icon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/src/icons/dhcp-icon.png</normaloff>:/icons/src/icons/dhcp-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>68</width>
         <height>68</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wifi24_icon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/src/icons/wifi-2.4-icon.png</normaloff>:/icons/src/icons/wifi-2.4-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>68</width>
         <height>68</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wifi5_icon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/src/icons/wifi-5.0-icon.png</normaloff>:/icons/src/icons/wifi-5.0-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>68</width>
         <height>68</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ipv6_icon">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/src/icons/ipv6-icon.png</normaloff>:/icons/src/icons/ipv6-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>68</width>
         <height>68</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="app_icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="suite_modal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>362</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>321</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="suite_label">
        <item>
         <widget class="QLabel" name="suite_id">
          <property name="font">
           <font>
            <pointsize>21</pointsize>
           </font>
          </property>
          <property name="text">
           <string>User Interface</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="add_all_icon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../resources.qrc">:/icons/src/icons/add-icon-.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="test_selector">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="fixer_modal">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>362</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="runner_btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>480</y>
       <width>150</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Run Test</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>321</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="fixer_label">
        <item>
         <widget class="QLabel" name="fixer_text">
          <property name="font">
           <font>
            <pointsize>21</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Test Fixer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rm_all_icon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../resources.qrc">:/icons/src/icons/remove-icon.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="test_fixer">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>test_fixer</zorder>
    <zorder>horizontalLayoutWidget_2</zorder>
    <zorder>runner_btn</zorder>
    <zorder>runner_btn</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuLogs">
    <property name="title">
     <string>Logs</string>
    </property>
    <addaction name="actionReports"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuLogs"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
